doc/extdev/builderapi.rst

```diff
@@ -13,6 +13,7 @@ Builder API
 
    .. graphviz:: /_static/diagrams/sphinx_build_flow.dot
       :caption: UML for the standard Sphinx build workflow
+         (method colors: green; abstract, orange; overridable, red; final)
```

Comment:
I fine this quite hard to understand. Maybe a paragraph below would be more helpful?

> The red methods define the fundamental workflow structure. They are builder-independent and therefore final. Builder-specific behavoir is implemented in abstract methods that every builder must implement (green) and methods with a default implmentation that can be overwritten if needed (orange).

An additional visual aid would be to be able to color the words "abstract method" in green etc. But I believe we don't have the capability for inline text coloring?

Alternatively / additionally, would it make sense to add a small legend
- final methods
- overridable methods
- abstract methods
to the graphviz itself? These could be colored accordingly.

Reply:
done 👍